### PR TITLE
player/command: don't return ill-formed node if there is no default val

### PR DESCRIPTION
**Pull request by kasper93**

### CODE REVIEW

**Comment by sfan5** [comment]
first commit is duplicate in #14542, consider merging that first.

**Comment by kasper93** [comment]
> first commit is duplicate in #14542, consider merging that first.

No, not yet.

File: player/command.c [hunk 1]
```diff
@@ -3605,7 +3605,8 @@ static int mp_property_option_info(void *ctx, struct m_property *prop,
 
         union m_option_value def = m_option_value_default;
```
Comment (sfan5) [inline player/command.c:3606]: could be null-initialized if it isn't meant to contain a valid value anyway

**Review by sfan5 — APPROVED** [review]
lgtm

File: player/command.c (hunk 1 above)
Comment (kasper93) [reply player/command.c:3606]: It is, `m_option_value_default` is null opt basically.

Comment (sfan5) [reply player/command.c:3606]: well either it's equivalent to `{0}` (then why does it exist?) or it's not.
but doesn't matter either way.

**Pull request merged**